src/playground/gilded-rose.ts

```diff
@@ -20,19 +20,19 @@ export class GildedRose {
   updateQuality() {
     for (let i = 0; i < this.items.length; i++) {
       if (
-        this.items[i].name != "Aged Brie" &&
-        this.items[i].name != "Backstage passes to a TAFKAL80ETC concert"
+        this.items[i].name !== "Aged Brie" &&
```

Comment:
Oh, the playground is intentionally invalid. We should exclude this folder from the linter. 

The playground code is the one that will open in the Extension Host when you debug locally.